shell/components/Tabbed/index.vue

```diff
@@ -105,7 +106,8 @@ export default {
   },
 
   data() {
-    const extensionTabs = this.showExtensionTabs ? getApplicableExtensionEnhancements(this, ExtensionPoint.TAB, TabLocation.RESOURCE_DETAIL, this.$route, this, this.extensionParams) || [] : [];
+    const location = this.isResourceDetailDrawer ? TabLocation.RESOURCE_SHOW_CONFIGURATION : TabLocation.RESOURCE_DETAIL;
```

Comment:
@aalves08 @codyrancher just want to confirm my comment in https://github.com/rancher/dashboard/issues/16189#issuecomment-3661177094.

The issue needs updating to reflect that we need it primarily on the edit page and then the show config tab, so two tab locations. 

We probably need to be more explicit as well (if a --> y, if b --> x) instead of falling back on a location (if then else). Tabs can be used in a number of different places